src/common/printprof.c

```diff
@@ -0,0 +1,763 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2014 pascal obry
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+
+    NOTE: this code is highly inspired from the tifficc tool from Little-CMS
+*/
+
+#include <glib.h>
+#include <unistd.h>
+#include "lcms2_plugin.h"
+#include "tiffio.h"
+#include "common/printprof.h"
+
+#define SW "-"
+
+typedef struct _print_profile_data_t
```

Comment:
i think you can actually give the struct the same name and the same tag. why the leading underscore?